hw/ip/keymgr/data/keymgr.hjson

```diff
@@ -331,6 +331,8 @@
     { name: "KMAC_IF_CMD.CTRL.CONSISTENCY",
       desc: '''
         One hot check for kmac interface commands.
+        Also, command enable (adv_en, id_en, gen_en) is checked for consistency
```

Comment:
the KEYMGR.KMAC_IF_CMD.CTRL.CONSISTENCY also does a consistency check [here](https://cs.opensource.google/opentitan/opentitan/+/master:hw/ip/keymgr/rtl/keymgr_kmac_if.sv;l=357?q=KMAC_IF_CMD.CTRL.CONSISTENCY%09%20&ss=opentitan%2Fopentitan:hw%2F) (during an operation, changing one type of operation to another one will trigger the error)